test/ably/restsetup.py

```diff
@@ -24,7 +24,7 @@
     host = "sandbox-rest.ably.io"
 
 if host.endswith("rest.ably.io"):
-    host = "sandbox-rest.ably.io"
+    host = "staging-rest.ably.io"
```

Comment:
Please don't commit this change as we want to continue to test against sandbox.  I realise it will make your test suite pass, but this will be fixed imminently and tests should always be run against sandbox by default